www/js/paths/pathsrepo.js

```diff
@@ -44,7 +44,7 @@ const landmarks2020 =
     rightmidLeft: Pose2d.fromXYTheta(Field.xsize-1, 0, 180),
     midmidRight: Pose2d.fromXYTheta(Field.xsize*.5, 0, 0),
     midmidLeft: Pose2d.fromXYTheta(Field.xsize*.5, 0, 180),
-};
```

Comment:
this semicolon is actually correct.  Different from a code-block, the definition of a javascript object is a formal statement.  Eg:

`let x = {a: 1, b:2};`